### PR TITLE
acquisition: add field total_amount in ES

### PR DESCRIPTION
**Pull request by vgranata**
Co-Authored-by: Valeria Granata <valeria@chaw.com>

## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## Dependencies

My PR depends on the following `rero-ils-ui`'s PR(s):

* rero/rero-ils-ui#<xx>

## How to test?

- What command should I have to run to test your PR?
- What should I test through the UI?

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?
- [ ] Cypress tests successful?

### CODE REVIEW

File: rero_ils/modules/acq_receipt_lines/listener.py [hunk 1]
```diff
@@ -40,3 +40,6 @@ def enrich_acq_receipt_line_data(sender, json=None, record=None, index=None,
             pid=record.order_line.account_pid,
             type='acac'
         )
+
+        json['total_amount'] = round(
+            record.amount*record.receipt.exchange_rate*record.quantity, 2)
```
Comment (zannkukai) [inline rero_ils/modules/acq_receipt_lines/listener.py:45]: My feeling is to place the least as possible logic into listener.
Maybe add a `@property` method into the `AcqReceiptLine` and call it to fill `json['total_amount']`

File: rero_ils/modules/acq_receipts/api.py [hunk 2]
```diff
@@ -166,6 +166,11 @@ def amount_adjustments(self):
         """Shortcut to get receipt amount adjustments."""
         return self.get('amount_adjustments', [])
 
+    @property
+    def exchange_rate(self):
+        """Shortcut to get receipt exchange_rate."""
+        return self.get('exchange_rate')
+
```
Comment (zannkukai) [inline rero_ils/modules/acq_receipts/api.py:173]: Is this method is covered by unitest ?

**Review by zannkukai — CHANGES_REQUESTED** [review]
I often feel like the bad guy who asks some non-essential changes :-) Sorry XD

File: rero_ils/modules/acq_receipts/api.py (hunk 2 above)
Comment (vgranata) [reply rero_ils/modules/acq_receipts/api.py:173]: OK, I just put it in the tests/ui

**Review by BadrAly — APPROVED** [review]

**Review by zannkukai — APPROVED** [review]

**Pull request merged**